index.css

```diff
@@ -0,0 +1,289 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+.blink {
+  color: #f8cb45;
+}
+.it {
+  color: #54b226;
+}
+.delivery {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid #f5f5f5;
+}
+.left,
+.right {
+  display: flex;
+  flex-direction: row;
+}
+.delivery .logo {
+  justify-content: flex-start;
+  padding: 10px 25px;
+  font-weight: 700;
+  border-right: 3px solid #f5f5f5;
+}
+.myCart {
+  padding: 10px;
+  margin-right: 30px;
+  color: #fff;
+  background-color: #23963f;
+  border: none;
+  border-radius: 5px;
+  margin-left: 30px;
+}
+.adanvatgesDesciption h3,
+.delivery .deliveryHead,
+.delivery .right {
```

Comment:
Create separate class for this styles and give some other name and reuse that at all 3 places